hil/cli/network.py

```diff
@@ -34,19 +36,64 @@ def network_delete(network):
 
 @network.command(name='show')
 @click.argument('network')
-def network_show(network):
+@click.option('--json', is_flag=True)
+def network_show(network, json):
     """Display information about network"""
     q = client.network.show(network)
-    for item in q.items():
-        sys.stdout.write("%s\t  :  %s\n" % (item[0], item[1]))
+    if json:
+        print q
+        return
+    net_table = PrettyTable()
+    net_table.field_names = ['Attribute', 'Info']
+
+    if 'owner' in q:
+        net_table.add_row(['owner', q['owner']])
```

Comment:
It might make it look neat if we capitalize the labels.
```python
net_table.add_row(['Owner', q['owner']])
```
but it's not something I'd block this on.